addons/website/static/tests/tours/configurator_translation.js

```diff
@@ -0,0 +1,64 @@
+odoo.define('test_website_modules.tour.configurator_translation', function (require) {
```

Comment:
`test_website_modules` :thinking: 

You can actually get rid of all that naming stuff if using the new way to declare JS files, As simple as that:

```
/** @odoo-module */

import tour from 'web_tour.tour';

tour.register('configurator_translation', {
```